bumblebee/modules/traffic.py

```diff
@@ -120,10 +140,19 @@ def _update_widgets(self, widgets):
                 name = "traffic.{}-{}".format(direction, interface)
                 widget = self.create_widget(widgets, name, attributes={"theme.minwidth": self.get_minwidth_str()})
                 prev = self._prev.get(name, 0)
-                speed = bumblebee.util.bytefmt(
-                    (int(data[direction]) - int(prev))/timediff,
-                    self._format)
+                bspeed = (int(data[direction]) - int(prev))/timediff
+                speed = bumblebee.util.bytefmt(bspeed, self._format)
                 txtspeed = '{0}/s'.format(speed)
+                if self._graphlen > 0:
+                    # skip first value returned by psutil, because it is
+                    # giant and ruins the grapth ratio until it gets pushed
+                    # out of saved list
+                    if self._first_run is True:
```

Comment:
maybe it would be more elegant to check for len(self._graphdata[interface][direction]) > 1 ?